src/04-modality-specific-files/05-task-events.md

```diff
@@ -165,7 +165,7 @@ The object supplied for `StimulusPresentation` SHOULD include the following key-
 | SoftwareName    | RECOMMENDED           | [string][]    | Name of the software that was used to present the stimuli.                                                                                                                                                       |
 | SoftwareRRID    | RECOMMENDED           | [string][]    | [Research Resource Identifier](https://scicrunch.org/resources) of the software that was used to present the stimuli. Examples: The RRID for Psychtoolbox is 'SCR_002881', and that of PsychoPy is 'SCR_006571'. |
 | SoftwareVersion | RECOMMENDED           | [string][]    | Version of the software that was used to present the stimuli.                                                                                                                                                    |
-| Code            | RECOMMENDED           | [string][]    | [URI][uri] of the code used to present the stimuli. Persistent identifiers such as DOIs are preferred. If multiple versions of code may be hosted at the same location, revision-specific URIs are recommended.  |
+| Code            | RECOMMENDED           | [string][]    | [URI](../02-common-principles.md#uniform-resource indicator) of the code used to present the stimuli. Persistent identifiers such as DOIs are preferred. If multiple versions of code may be hosted at the same location, revision-specific URIs are recommended.  |
```

Comment:
I suspect that I should actually use the `link definitions` section at the bottom of each document even when it comes to internal links, right?

Reply:
The `link definitions` section has two advantages: 

1. links that are repeating in the document only have to be defined once
1. the reading flow of the markdown source text is not interrupted by long links (this is especially useful for formatting markdown tables, where you don't want to have too long columns just due to links)

we currently don't have strict guidelines on when to use which kind of link.